### ISSUE TITLE
Support for attaching multiple target groups to one ECS service

### ISSUE THREAD

**Issue by poflynn** [terraform-providers/terraform-provider-aws#9571]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Previously only one target group could point to one ECS service at a time. Amazon now supports having multiple target groups pointing to a single ECS service

### New or Affected Resource(s)

* aws_ecs_service

### Potential Terraform Configuration

I would imagine code might look like this. Note that in the "load_balancer" section the "target_group_arn" is a list and not a single item like it is today altho this might cause backward compatibility issues so maybe isn't the right answer.

```hcl
resource "aws_ecs_service" "example_ecs" {
  name                               = "example-ecs"
  task_definition                    = "${aws_ecs_task_definition.service_task_def.id}"
  desired_count                      = "3"
  deployment_minimum_healthy_percent = "50"

  cluster  = "cluster-ecs"
  iam_role = "${aws_iam_role.ecs_service_role.arn}"

  load_balancer {
    target_group_arn = "[
      ${aws_lb_target_group.target_group1.arn}, 
      ${aws_lb_target_group.target_group2.arn},
    ]"
    container_name   = "container_name"
  }
```

### References

See post below for more detail (dated July 30, 2019).
https://aws.amazon.com/about-aws/whats-new/2019/07/amazon-ecs-services-now-support-multiple-load-balancer-target-groups/

**Comment by bflad** [comment]
Hi @poflynn 👋 Support for multiple target groups (via multiple `load_balancer` configuration blocks) has already been merged (#9411) and will release with [version 2.22.0 of the Terraform AWS Provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2220-unreleased), tomorrow. 👍

**Comment by poflynn** [comment]
Ah, I did a search only of open issues, it never occurred to me that it would have already been closed! Thanks!! @bflad
